tests/TestBaseClassWeb.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ *  LimeSurvey
+ * Copyright (C) 2007-2011 The LimeSurvey Project Team / Carsten Schmitz
+ * All rights reserved.
+ * License: GNU/GPL License v2 or later, see LICENSE.php
+ * LimeSurvey is free software. This version may have been modified pursuant
+ * to the GNU General Public License, and as distributed it includes or
+ * is derivative of works licensed under the GNU General Public License or
+ * other free or open source software licenses.
+ * See COPYRIGHT.php for copyright notices and details.
+ */
+
+namespace ls\tests;
+
+use Facebook\WebDriver\Remote\DesiredCapabilities;
+use Facebook\WebDriver\Remote\RemoteWebDriver;
+use Facebook\WebDriver\WebDriver;
+use Facebook\WebDriver\WebDriverBy;
+use Facebook\WebDriver\WebDriverExpectedCondition;
+
+/**
+ * Class TestBaseClassWeb
+ * this is the base class for functional tests that need browser simulation
+ * @package ls\tests
+ */
+class TestBaseClassWeb extends TestBaseClass
+{
+    /**
+     * @var int web server port
+     * TODO this should be in configuration somewhere
```

Comment:
Again problem with local phpunit.xml vs phpunit.xml in repository. For domain, I used command line environment. Could do the same for port of phantomjs. Too many command line arguments in the end?

Reply:
I also kind of do not like the command line arguments. 
what about having a phpunit.xml.example and a gitignored phpunit.xml with modifiable config. This could also have the test DB stuff?

Reply:
Yes, but then it would be harder to push changes into the xml. The best would be to use _two_ phpunit.xml files, but could not find any support for that. :(